Lib/test/test_interpreters/test_channels.py

```diff
@@ -120,12 +120,13 @@ def test_equality(self):
         self.assertNotEqual(ch1, ch2)
 
     def test_pickle(self):
-        ch, _ = channels.create()
-        for protocol in range(2, pickle.HIGHEST_PROTOCOL + 1):
-            with self.subTest(protocol=protocol):
-                data = pickle.dumps(ch, protocol)
-                unpickled = pickle.loads(data)
-                self.assertEqual(unpickled, ch)
+        recv, send = channels.create()
+        for ch in [recv, send]:
```

Comment:
This test repeats also the test in TestSendChannelAttrs. I think that only needs to test the receiver channel here.
```suggestion
        ch, _ = channels.create()
```
(and de-indent the following code)

Reply:
Thanks, changed!